components/organisms/Contributors/contributors.tsx

```diff
@@ -51,6 +79,103 @@ const Contributors = ({ repositories }: ContributorProps): JSX.Element => {
         };
       });
 
+  const PopOverListContent = () => {
+    const { data } = useFetchAllLists();
+    const [loading, setLoading] = useState(false);
+
+    const handleAddContributorsToList = async () => {
+      if (!selectedListIds.length) {
+        return;
+      }
+
+      setLoading(true);
+      const response = Promise.all(
+        selectedListIds.map((listIds) =>
+          addListContributor(
+            listIds,
+            selectedContributors.map((contributor) => contributor.user_id)
+          )
+        )
+      );
+      response
+        .then((res) => {
+          toast({
+            description: `Successfully added ${selectedContributors.length} contributors to ${selectedListIds.length} lists!`,
+            variant: "success",
+          });
+        })
+        .catch((res) => {
+          toast({
+            description: `
+            An error occurred while adding contributors to lists. Please try again later.
```

Comment:
wording (non-blocking): Saying try again is probably enough. No need for the later.

```suggestion
            description: `
            An error occurred while adding contributors to lists. Please try again.
```